imports/ui/lists/recentAgents.coffee

```diff
@@ -26,9 +27,20 @@ Template.recentAgents.onCreated ->
           if binding.days.value > 30
             binding.dm = true
         @recentAgents.insert(binding)
+        if @recentAgents.find(articleId: articleId).count() is 6
+          @articles.update(articleId, { $set: { collapsed: true } })
 
 Template.recentAgents.helpers
   articles: ->
     Template.instance().articles.find()
-  recentAgentsForArticle: (articleId) ->
-    Template.instance().recentAgents.find(articleId: articleId)
+  isCollapsed: (articleId) ->
+    Template.instance().articles.findOne(articleId).collapsed
+  recentAgentsForArticle: (articleId, limit) ->
+    options = { sort: { 'days.value': 1 } }
```

Comment:
The sort order is backwards. We should show the least recently mentioned terms first. The terms with no prior mentions are correctly shown on top, but terms with a prior mention are being shown with the most recently mentioned term first.